mmv1/products/vertexai/Model.yaml

```diff
@@ -309,9 +318,9 @@ properties:
     ignore_read: true
   - type: Array
     name: 'versionAliases'
```

Comment:
After some further investigation we may want to consider removing this for the time being, there is currently no way to include suppot of `StateFunc` which would help with supporting this (more can be found [here](https://github.com/GoogleCloudPlatform/magic-modules/pull/10936#issuecomment-2168954728)). A Diff_suppress_func can't be applied due to lack of support on Lists.

I was almost convinced to leave it with `ignore_read: true` though this leaves us with a case where we don't receive the version_aliases when copying (e.g the target model contains v1beta1 but instead is set as null when attempting to copy)

Because of the reasons above it may be best to remove this field. Any thoughts on this would be appreciated.